src/rawecdsa.cc

```diff
@@ -8,7 +8,10 @@
 #include "openssl/pem.h"
 #include "openssl/x509.h"
 #include "v8.h"
+#include <string.h>
```

Comment:
Move this to the very top of the file and separate it from the `#include "node.h"` by a newline.